httplug/usage.rst

```diff
@@ -0,0 +1,12 @@
+HTTPlug usage
+=============
+
+HTTPlug is relevant for three groups of users:
+
+.. toctree::
+   :maxdepth: 1
+
+   Library users <users>
+   Application developers <application-developers>
+   Library developers <library-developers>
```

Comment:
The terms are explained more in the relevant chapters. Library = reusable package; do you prefer the term reusable ‘package’ over ‘library’?

Developers = those that create the library/package.
Users = those that include the library/package in their application.